### PR TITLE
Implement basic operation of class static field

### PR DESCRIPTION
**Pull request by ksh8281**
TODO: implement super keyword of static field init

### CODE REVIEW

File: src/parser/esprima_cpp/esprima.cpp [hunk 1]
```diff
@@ -5406,15 +5449,55 @@ class Parser {
...
+                    auto ch = this->scanner->sourceCharAt(idx);
+                    if (EscargotLexer::isLineTerminator(ch)) {
+                        hasLineBreak = true;
+                        break;
+                    }
+                    if (!EscargotLexer::isWhiteSpace(ch)) {
+                        break;
+                    }
+                    idx--;
+                }
```
Comment (clover2123) [inline src/parser/esprima_cpp/esprima.cpp:5484]: I'm wondering why this part is inserted.
Could you explain it more?

Comment (ksh8281) [reply src/parser/esprima_cpp/esprima.cpp:5484]: if there is no semicolon like
```
class C {
static a
static b = 1;
}
```
we should process even there is no semicolon like there is a semicolon

**Review by bbrto21 — APPROVED** [review]

File: src/parser/esprima_cpp/esprima.cpp (hunk 1 above)
Comment (clover2123) [reply src/parser/esprima_cpp/esprima.cpp:5484]: I see.
But what if there is a comment between no-semicolon static field and next field like below?
Is it still illegal expression or do we have to deal with this case too?
```js
class C {
  static a
  /* comment */ static b = 1;
}
```

Comment (ksh8281) [reply src/parser/esprima_cpp/esprima.cpp:5484]: You are right. I would consider that case. I will change this pr

Comment (ksh8281) [reply src/parser/esprima_cpp/esprima.cpp]: I updated pr

Comment (clover2123) [reply src/parser/esprima_cpp/esprima.cpp]: Looks great!

**Review by clover2123 — APPROVED** [review]

**Pull request merged**